src/cascadia/TerminalSettingsEditor/Launch.cpp

```diff
@@ -6,6 +6,7 @@
 #include "Launch.g.cpp"
 #include "LaunchPageNavigationState.g.cpp"
 #include "EnumEntry.h"
+#include "Utils.h"

```

Comment:
This change is a bit odd. Curious, why?

Reply:
We don't need this one after all, turns out it's in the headers for most of these
```suggestion
```